data/json/items/comestibles.json

```diff
@@ -8142,9 +8142,9 @@
     },
     {
         "type" : "COMESTIBLE",
-        "id": "hickory_nut_unshelled",
-        "name": "handful of unshelled hickory nuts",
-        "name_plural": "handfuls of unshelled hickory nuts",
+        "id": "hickory_nut_shelled",
+        "name": "handful of shelled hickory nuts",
```

Comment:
The only things you need to (or for that matter the only things you CAN) change are "name" and "name_plural".